pkg/runner/cluster_k8s.go

```diff
@@ -288,6 +283,13 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.W
 		return nil, err
 	}
 
+	runName := fmt.Sprintf("compress-tg-%s-%s", input.TestPlan.Name, input.RunID)
```

Comment:
Note that this part runs only when the run has `completed` (i.e. it is successful, or it has timed-out).

I imagine that devs will be killing Testground for a while, until their testplan is correct, so we should probably have a way to compress and propagate back the outputs on a canceled run as well.